sentry_wrapper/__init__.py

```diff
@@ -59,15 +95,42 @@ def execute():
         'group', nargs='?', default='console_scripts',
         help='Entry point group (default: console_scripts)'
     )
-    parser.add_argument(
-        '--dsn', metavar='SENTRY_DSN', default=os.getenv('SENTRY_DSN'),
-        help='Sentry DSN'
-    )
     parser.add_argument(
         '-t', '--timeout', metavar='timeout',
         type=int,
         help='Timeout. After this value, TimeoutError is raised to Sentry.'
     )
+    parser.add_argument(
+        '--dsn', metavar='SENTRY_DSN', default=os.getenv('SENTRY_DSN'),
+        help='Sentry DSN',
+    )
+    parser.add_argument(
+        '--release',
+        nargs='?',
+        help="The program's release version (e.g. 3.2.3)",
+    )
+    parser.add_argument(
+        '--environment',
+        nargs='?',
+        help='The envionment the program is running in (e.g. staging)',
+    )
+    parser.add_argument(
```

Comment:
IIRC the `server_name` is automatically deduced from `socket.gethostname()` ([doc](https://docs.sentry.io/clients/python/advanced/)).

In our Go stack (cf. `go-api-tools`) the configuration variable `ServerName` has been deprecated in favor of `ServiceName`. Maybe we should do the same for the Python stack?

Reply:
I just added the options available for the python configuration: https://docs.sentry.io/platforms/python/configuration/options/#server-name.
We don't have to use and probably won't but at least we have the possibility. I don't see any `serivce_name` option in the doc.